starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -199,7 +205,7 @@ def pre_deployed_account_with_validate_deploy(
         address=address,
         client=FullNodeClient(node_url=network + "/rpc"),
         key_pair=KeyPair.from_private_key(int(private_key, 16)),
-        chain=StarknetChainId.GOERLI,
+        chain=StarknetChainId.MAINNET,
```

Comment:
is that right? maybe I misunderstand something, but this may be sepolia in case of network tests, right? so this might fail?

Reply:
I think we use this function only with devnet, maybe simplify it. Another solution, after the merge https://github.com/software-mansion/starknet.py/pull/1327 can use the option without setting chain id so it will be taken from `get_chan_id`

Reply:
I think there are more places with similar logic. Probably all things related to `--net` parameter could be simplified. I create an issue for it. WDYT?

Reply:
let's briefly look through the code and try to assess it first. My suspicion is that there won't be that many places tbh

Reply:
I remove it and tests works https://github.com/software-mansion/starknet.py/pull/1328/commits/c306b0077e67a9e47a9b6219dbbdfe7d415cc5f6

Reply:
ok, so after this was removed there is no way to run this test on network. should we have this test for sepolia as well, or was it something that is non-essential/tested in some other way?
@ddoktorski also keen to hear what you have to say about this

applicable to all the places that changes this